app/models/miq_expression/relative_datetime.rb

```diff
@@ -37,7 +31,7 @@ def self.normalize(rel_time, tz, mode = "beginning")
       interval = interval.pluralize
 
       if interval == "hours"
-        beginning_or_end_of_hour(value.to_i.hours.ago.in_time_zone(tz), mode)
+        value.to_i.hours.ago.in_time_zone(tz).public_send("#{mode}_of_hour")
```

Comment:
It does.....we are using `send` elsewhere but no reason to not choose the above while I'm working on this. Consider it done!